requirements3.txt

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
So which python3 is that now? The output from `pip-compile` might be different depending on the python minor version that was used...

Reply:
I'm not sure there is a difference between 3.5, 3.6, 3.7 however if there a big difference we can add corresponding file. I was trying to generate it on 3.6 and 3.7 it seemed to me it produced the same content. For now we are resolving more global difference is 2 and 3.

Reply:
ok. good. then let's keep it simple. thanks for checking!

Reply:
btw, the jenkins.ceph.com slave is running python3.5 and it works fine so far